external/mqtt/src/test/scala/org/apache/spark/streaming/mqtt/MQTTStreamSuite.scala

```diff
@@ -113,7 +115,8 @@ class MQTTStreamSuite extends FunSuite with Eventually with BeforeAndAfter {
   }
 
   private def findFreePort(): Int = {
```

Comment:
I wonder if this can be refactored rather than duplicated? don't bother unless there is clearly a good common home for it in test code. I don't know if there is.